### PR TITLE
Add Slime Mori Mori Dragon Quest 3

### PR DESCRIPTION
**Pull request by CakeLancelot**
Adds Slime Mori Mori Dragon Quest 3.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by CakeLancelot** [comment]
I've created the wiki page [here](https://github.com/citra-emu/citra-games-wiki/wiki/Slime-Mori-Mori-Dragon-Quest-3). 

Do note that I intended to submit the exact Japanese title (スライムもりもりドラゴンクエスト３) as the form specified, but received an error when I did so (see below). So I substituted an English translation and that seemed to work.

```Error message: Unprocessable Entity
Type: undefined
Current Task: 12
Task Count: 16
Release/Screenshot/Savefile Count: 6
State:
	trees: [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]
	username: CakeLancelot
	toml: [object Object]
	gameSlug: 
	shaSum: 4b43168f8863e3c7b0b19789b8fcb3ba94fb2e91
	repoName: CakeLancelot/citra-games-wiki
```

File: games/slime-mori-mori-dragon-quest-3/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "000400000005C300"
+region = "jpn"
+release_date = "2011-11-02"
+
+[[testcases]]
+title = "000400000005C300"
+compatibility = "1"
+date = "2017-11-08"
+version = "HEAD-31c0d4c"
```
Comment (Leo626) [inline games/slime-mori-mori-dragon-quest-3/game.dat]: Testcases should only be done with Nightly.

File: games/slime-mori-mori-dragon-quest-3/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Slime Mori Mori Dragon Quest 3"
+description = "Spinoff game of the Dragon Quest series, you control a slime that must sail the sea and reclaim the stolen rainbow orbs. Contains a 'zelda' like adventuring section as well as a ship battle system."
```
Comment (TheKoopaKingdom) [inline games/slime-mori-mori-dragon-quest-3/game.dat]: The beginning sounds a bit awkward, "Spinoff game of the Dragon Quest series, **where** you control..." sounds better.

File: games/slime-mori-mori-dragon-quest-3/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+region = "jpn"
+release_date = "2011-11-02"
+
+[[testcases]]
+title = "000400000005C300"
+compatibility = "1"
+date = "2017-11-08"
+version = "HEAD-31c0d4c"
+cpu = "AMD Ryzen 5 1600"
+gpu = "Radeon R9 Fury"
```
Comment (TheKoopaKingdom) [inline games/slime-mori-mori-dragon-quest-3/game.dat]: Prepend `AMD` to the name of the GPU.

**Comment by CakeLancelot** [comment]
I've updated the game.dat and wiki, should be correct now.

**Comment by Leo626** [comment]
The wiki should follow something similar to the example in the [Contributing](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#wiki) guide. It should also be describing any graphical issues that might be observed.

**Comment by CakeLancelot** [comment]
> The wiki should follow something similar to the example in the Contributing guide. It should also be describing any graphical issues that might be observed.

I didn't observe any during my testing, which is why it wasn't mentioned. I've changed it now to mention this and bring it more in-line with the contribution guidelines.

**Pull request merged**